tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -47,6 +49,20 @@ internal TracerManager CreateTracerManager(ImmutableTracerSettings settings, Tra
                 runtimeMetrics: null,
                 logSubmissionManager: previous?.DirectLogSubmission,
                 telemetry: null);
+
+            try
+            {
+                NativeInterop.SetApplicationInfoForAppDomain(RuntimeId.Get(), tracer.Settings.ServiceName, tracer.Settings.Environment, tracer.Settings.ServiceVersion);
```

Comment:
should't we use `tracer.DefaultServiceName` instead ?
This will allows us to benefit from the service name computation (IIS, AAS...) done at line 84

Reply:
Right, good point